loader/src/sources/wa-doh.js

```diff
@@ -65,6 +66,16 @@ const LOCATIONS_QUERY = `
   }
 `;
 
+function warn(message, context) {
+  console.warn(`WA DoH: ${message}`, context);
+  // Sentry does better fingerprinting with an actual exception object.
+  if (message instanceof Error) {
+    Sentry.captureException(message, { level: Sentry.Severity.Info });
+  } else {
+    Sentry.captureMessage(message, Sentry.Severity.Info);
+  }
+}
```

Comment:
At this point, we obviously need to generalize this. I don’t think I want to do that in this PR, though.